spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -7,18 +7,19 @@
 
 RSpec.describe Reek::Configuration::ConfigurationFileFinder do
   describe '.find' do
-    it 'returns the path if it’s set' do
+    it 'returns the file in :path if it’s set' do
```

Comment:
I wanted to make it clearer that it would use the path argument. Maybe reword to `the argument in :path`?